en/docs/learn/adding-users-and-roles.md

```diff
@@ -0,0 +1,93 @@
+# Adding Users and Roles
+
+## Scenario
+This tutorial demonstrates user and role management in WSO2 Identity Server. User management involves defining and managing users, roles, and access levels in a system. A typical user management implementation involves a wide range of functionality such as adding/deleting users, controlling user activity through permissions, and managing user roles.  
+
+This tutorial demonstrates how to create a new user, create a role with a set of permissions, and then log in to the dashboard as the newly created user. 
+
+## Setting up
+
+1. [Download WSO2 Identity Server](https://wso2.com/identity-and-access-management/).
+
+2. Navigate to `<IS_HOME>/bin` and start the server by executing one of the following commands.
+
+    ``` java tab="Linux/MacOS"
+    sh wso2server.sh
+    ```
+
+    ``` java tab="Windows"
+    wso2server.bat run
+    ```
+
+3. Access the [WSO2 Identity Server Management Console] (https://localhost:9443/carbon) and log in using "admin" as both the username and password. 
```

Comment:
Access the [WSO2 Identity Server Management Console] (https://localhost:9443/carbon) as an administrator (credentials: `admin`/`admin`).